### PR TITLE
Add A New Idea Discussion and remove github finder

### PR DESCRIPTION
**Pull request by Programming-School-Pro-Coding**
> First thing, PLEASE READ THIS: [ReactPlay Code Review Checklist](https://github.com/atapas/react-play/wiki/ReactPlay-Code-Review-Checklist)

# Description

Add A New Idea

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. 

# Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by Programming-School-Pro-Coding** [comment]
@atapas

File: .github/FUNDING.yml [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # These are supported funding model platforms
 
-custom: ["https://blog.greenroots.info/sponsor", "https://buymeacoffee.com/greenroots", "https://paypal.me/greenrootsinfo"]
```
Comment (atapas) [inline .github/FUNDING.yml:3]: This file do not need changes

File: .github/ISSUE_TEMPLATE/----bug-report---.md [hunk 2]
```diff
@@ -35,4 +35,4 @@ If applicable, add screenshots to help explain your problem.
  - Version [e.g. 22]
 
 **Additional context**
-Add any other context about the problem here.
```
Comment (atapas) [inline .github/ISSUE_TEMPLATE/----bug-report---.md:38]: This file do not need changes

File: .github/ISSUE_TEMPLATE/---feature-request---.md [hunk 3]
```diff
@@ -17,4 +17,4 @@ A clear and concise description of what you want to happen.
 A clear and concise description of any alternative solutions or features you've considered.
 
 **Additional context**
-Add any other context or screenshots about the feature request here.
```
Comment (atapas) [inline .github/ISSUE_TEMPLATE/---feature-request---.md:20]: This file do not need changes

File: .github/ISSUE_TEMPLATE/new-play.md [hunk 4]
```diff
@@ -15,4 +15,4 @@ Thank You!
 ## What ReactJS concept will be used to create this play?
 
 
-## Are you willing to take it up for implementation?
```
Comment (atapas) [inline .github/ISSUE_TEMPLATE/new-play.md:18]: This file do not need changes

File: .github/pull_request_template.md [hunk 5]
```diff
@@ -1,3 +1,6 @@
+
+> First thing, PLEASE READ THIS: [ReactPlay Code Review Checklist](https://github.com/atapas/react-play/wiki/ReactPlay-Code-Review-Checklist)
+
 # Description
 
 Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
```
Comment (atapas) [inline .github/pull_request_template.md:6]: This file do not need changes

File: package.json [hunk 6]
```diff
@@ -16,6 +16,7 @@
     "react-router-dom": "6",
     "react-scripts": "5.0.0",
     "react-shimmer-effect": "^1.0.9",
+    "styled-components": "^5.3.5",
```
Comment (atapas) [inline package.json:19]: Why is this added

File: package.json [hunk 7]
```diff
@@ -51,6 +52,7 @@
     ]
   },
   "devDependencies": {
+    "@types/styled-components": "^5.1.25",
```
Comment (atapas) [inline package.json:55]: Why is this added?

**Review by atapas — CHANGES_REQUESTED** [review]

File: package.json (hunk 6 above)
Comment (Programming-School-Pro-Coding) [reply package.json:19]: I added it when i made github finder

File: package.json (hunk 7 above)
Comment (Programming-School-Pro-Coding) [reply package.json:55]: I added it when i made github finder

Comment (atapas) [reply package.json]: Ok remove it, now not needed

File: package.json (hunk 6 above)
Comment (atapas) [reply package.json]: Not needed now

Comment (Programming-School-Pro-Coding) [reply package.json]: Ok I fix that

**Comment by Programming-School-Pro-Coding** [comment]
@atapas please review

**Comment by atapas** [comment]
@Programming-School-Pro-Coding  Reviewed. It still has changes to many files unnecessarily. Please check the diffs: https://github.com/atapas/react-play/pull/220/files

I really the changes should be in one file PlayIdeas. Could you please revert changes to other files?

**Comment by atapas** [comment]
Hello @Programming-School-Pro-Coding 

We will be moving the react-play project from the `atapas` account to an organization to help manage the project in a better way. The new organization URL is: https://github.com/reactplay.

> **The movement will take place between 12:30 - 14:00 hrs IST(7:00 am - 8:30 am GMT), 30th May 2022. Please refrain from pushing any changes between this time.**

We will inform you when the movement is complete. You need to make a few minimal configuration changes after that. We will let you know.

**Comment by Programming-School-Pro-Coding** [comment]
> @Programming-School-Pro-Coding Reviewed. It still has changes to many files unnecessarily. Please check the diffs: https://github.com/atapas/react-play/pull/220/files
> 
> The changes should be in one file PlayIdeas. Could you please revert changes to other files?

Done

**Comment by Programming-School-Pro-Coding** [comment]
> ### 👷 Deploy Preview for _reactplayio_ processing.
> Name	Link
> 🔨 Latest commit	[5e71f6d](https://github.com/reactplay/react-play/commit/5e71f6da4a26495251d2aaeaf52e0267e6ff6635)
> 🔍 Latest deploy log	https://app.netlify.com/sites/reactplayio/deploys/6294a0ae58074d0008383bef

Ok

**Comment by atapas** [comment]
> Hello @Programming-School-Pro-Coding
> 
> We will be moving the react-play project from the `atapas` account to an organization to help manage the project in a better way. The new organization URL is: https://github.com/reactplay.
> 
> > **The movement will take place between 12:30 - 14:00 hrs IST(7:00 am - 8:30 am GMT), 30th May 2022. Please refrain from pushing any changes between this time.**
> 
> We will inform you when the movement is complete. You need to make a few minimal configuration changes after that. We will let you know.

The migration is complete. You can find the project here: https://github.com/reactplay

You no need to reclone the project. You just need to set the correct upstream. Here is the link: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/configuring-a-remote-for-a-fork

**Comment by Programming-School-Pro-Coding** [comment]
> > Hello @Programming-School-Pro-Coding
> > We will be moving the react-play project from the `atapas` account to an organization to help manage the project in a better way. The new organization URL is: https://github.com/reactplay.
> > > **The movement will take place between 12:30 - 14:00 hrs IST(7:00 am - 8:30 am GMT), 30th May 2022. Please refrain from pushing any changes between this time.**
> > 
> > 
> > We will inform you when the movement is complete. You need to make a few minimal configuration changes after that. We will let you know.
> 
> The migration is complete. You can find the project here: https://github.com/reactplay
> 
> You no need to reclone the project. You just need to set the correct upstream. Here is the link: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/configuring-a-remote-for-a-fork

Ok, Please complete review

**Comment by atapas** [comment]
> > @Programming-School-Pro-Coding Reviewed. It still has changes to many files unnecessarily. Please check the diffs: https://github.com/atapas/react-play/pull/220/files
> > The changes should be in one file PlayIdeas. Could you please revert changes to other files?
> 
> Done

@Programming-School-Pro-Coding You have deleted the files. That's not the right way. If we merge this PR, it will delete those files on main which is incorrect. You had to revert the changes instead.

If the revert is tricky, I would suggest to discard this PR and branch. Create a new branch, do the changes and create a PR.

**Comment by Programming-School-Pro-Coding** [comment]
Ok, Please a moment

**Comment by Programming-School-Pro-Coding** [comment]
@atapas Review Again

**Comment by Programming-School-Pro-Coding** [comment]
@atapas

**Review by atapas — CHANGES_REQUESTED** [review]
@Programming-School-Pro-Coding Ok let me try explaining it again.

Your changes should be in only 1 file. But if you see, the file changes shows in 5 files. 
image

Other 4 file changes are absolutely not necessary. You may not be syncing your branch from upstream which could be causing this issue.

**Comment by Programming-School-Pro-Coding** [comment]
> @Programming-School-Pro-Coding Ok let me try explaining it again.
> 
> Your changes should be in only 1 file. But if you see, the file changes shows in 5 files. image
> 
> Other 4 file changes are absolutely not necessary. You may not be syncing your branch from upstream which could be causing this issue.

You erased .Accidentally attracted Folder

**Comment by Programming-School-Pro-Coding** [comment]
@atapas

**Comment by atapas** [comment]
> > @Programming-School-Pro-Coding Ok let me try explaining it again.
> > Your changes should be in only 1 file. But if you see, the file changes shows in 5 files. image
> > Other 4 file changes are absolutely not necessary. You may not be syncing your branch from upstream which could be causing this issue.
> 
> I deleted .github folder Accidentally

I suggest you close this PR, create fresh new branch from main, do that 1 file change and create a new PR. Tag me in that once you done.

**Comment by Programming-School-Pro-Coding** [comment]
> > > @Programming-School-Pro-Coding Ok let me try explaining it again.
> > > Your changes should be in only 1 file. But if you see, the file changes shows in 5 files. image
> > > Other 4 file changes are absolutely not necessary. You may not be syncing your branch from upstream which could be causing this issue.
> > 
> > 
> > I deleted .github folder Accidentally
> 
> I suggest you close this PR, create fresh new branch from main, do that 1 file change and create a new PR. Tag me in that once you done.

Closed

**Pull request closed without merging**